lib/license_scout/dependency_manager/dep.rb

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright:: Copyright 2016, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "ffi_yajl"
+require "yaml"
+require "toml-rb"
+require "license_scout/dependency_manager/base"
+
+module LicenseScout
+  module DependencyManager
+    class Dep < Base
```

Comment:
To the uninitiated, `godep` vs `dep` is pretty confusing, can you add a comment to clarify that this refers to this one: https://github.com/golang/dep Thanks!

Reply:
Agree, thanks